pkg/pdatatest/plogtest/logs.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package comparetest // import "github.com/open-telemetry/opentelemetry-collector-contrib/internal/comparetest"
+package plogtest // import "github.com/open-telemetry/opentelemetry-collector-contrib/pkg/pdatatest/plogtest"
```

Comment:
`CompareLogs` is redundant? Just Compare is good enough, or you think of adding things like CompareLogRecord?

Reply:
We already have Compare to other objects here like `CompareScopeLogs` and  `CompareLogRecords`